.github/workflows/npm-meteor-promise.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         working-directory: npm-packages/meteor-promise
     strategy:
       matrix:
-        node-version: [14.x, 20.x]
+        node-version: [12.x, 14.x]
```

Comment:
If this is being reversed for Fibers, then 14.x is good enough. No need to run another test for 12.x.